open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/SubjectAreaEntityClient.java

```diff
@@ -11,14 +11,39 @@
 
 import java.util.List;
 
+/**
+ * @param <E> inherited from {@link Node} object type
```

Comment:
I would prefer not talking about types in a API too much - can we remove object type. Conceptually Egeria has a type system -which this API is trying to hide. We are talking here about the types in the java api , but it could be confusing.

Reply:
No developer can be confused by this description.) And in this case, javadoc requires a generic type description from me.